pico/runtime/src/main/java/io/helidon/pico/runtime/Dependencies.java

```diff
@@ -24,21 +24,22 @@
 
 import io.helidon.common.types.DefaultTypeName;
 import io.helidon.common.types.TypeName;
-import io.helidon.pico.DefaultDependenciesInfo;
-import io.helidon.pico.DefaultDependencyInfo;
-import io.helidon.pico.DefaultInjectionPointInfo;
-import io.helidon.pico.DefaultQualifierAndValue;
-import io.helidon.pico.DefaultServiceInfoCriteria;
-import io.helidon.pico.DependenciesInfo;
-import io.helidon.pico.DependencyInfo;
-import io.helidon.pico.ElementInfo;
-import io.helidon.pico.InjectionPointInfo;
-import io.helidon.pico.QualifierAndValue;
-import io.helidon.pico.ServiceInfoCriteria;
+import io.helidon.pico.api.Activator;
+import io.helidon.pico.api.DefaultDependenciesInfo;
+import io.helidon.pico.api.DefaultDependencyInfo;
+import io.helidon.pico.api.DefaultInjectionPointInfo;
+import io.helidon.pico.api.DefaultQualifierAndValue;
+import io.helidon.pico.api.DefaultServiceInfoCriteria;
+import io.helidon.pico.api.DependenciesInfo;
+import io.helidon.pico.api.DependencyInfo;
+import io.helidon.pico.api.ElementInfo;
+import io.helidon.pico.api.InjectionPointInfo;
+import io.helidon.pico.api.QualifierAndValue;
+import io.helidon.pico.api.ServiceInfoCriteria;
 
 /**
  * This is the class the code-generator will target that will be used at runtime for a service provider to build up its
- * dependencies expressed as {@link io.helidon.pico.DependenciesInfo}.
+ * dependencies expressed as {@link DependenciesInfo}.
```

Comment:
We should use fully qualified links in javadoc, so if there is only a javadoc reference, there is no import in class

Reply:
next push

Reply:
pushed, as well as a whole bunch more in this category